public/src/main/java/com/revenuecat/purchases/models/StoreProduct.kt

```diff
@@ -46,15 +46,15 @@ data class StoreProduct(
     /**
      * Formatted original price of the item, including its currency sign.
      *
-     * Note: returned only for Google products.
+     * Note: returned only for Google products. Not available for Amazon.
      */
     val originalPrice: String?,
 
     /**
      * Returns the original price in micro-units, where 1,000,000 micro-units equal one unit
      * of the currency.
      *
-     * Note: returned only for Google products.
+     * Note: returned only for Google products. Not available for Amazon.
      */
     val originalPriceAmountMicros: Long,
```

Comment:
We might want to update to say `0 for Amazon subscriptions` like the other `Long`s